accname/name/comp_text_node.html

```diff
@@ -13,23 +13,96 @@
 
 <p>Tests the <a href="https://w3c.github.io/accname/#comp_text_node">#comp_text_node</a> portions of the AccName <em>Name Computation</em> algorithm.</p>
 
-<!-- I'm not certain whether #comp_text_node requires a lot of testing outside of the #comp_name_from_content contexts, -->
-<!-- but I did think of one example where text node versus comment node may make a difference when joining text nodes with a space vs innerText. -->
+<!-- Note: some overlap with the tests in ./comp_name_from_content.html -->
```

Comment:
Additional tests I would propose:
- Text nodes as part of explicit ARIA roles
- Tests that ensure labelledby/aria-label/name from content etc. take precedence over text nodes (may be captured in respective files)

Reply:
> * Text nodes as part of explicit ARIA roles

I will add this. Assuming this means `<div role="heading">` and `<div role="button" tabindex="0">` in addition to `<h2>` and `<button>`.

> * Tests that ensure labelledby/aria-label/name from content etc. take precedence over text nodes (may be captured in respective files)

I think the ones you wrote in #41463 test most of that, and more will come in https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/68 once unblocked.

Reply:
Resolved in https://github.com/web-platform-tests/wpt/pull/42407/commits/260d5e5354a86017d37439503660b90f85c1c856